src/generate_stochastic_block_model_with_features.py

```diff
@@ -25,7 +25,11 @@
 
 import numpy as np
 
+<<<<<<< HEAD
```

Comment:
This merge boilerplate should be removed. Also, both the mwe and local test runs should have failed if these lines are here. Did you test either the mwe or local test runs after this last commit?

Reply:
@jpalowitch Apologies for missing out on that! Afaik the main_local and main_local_mwe runs don't call this file at any point and it is just a standalone test script? 
Mentioning as both runs worked fine when testing that commit and I can't find an entrypoint into the file from the main runs.